From: kexinsun <kexinsun@smail.nju.edu.cn>
Subject: mm/hugetlb_vmemmap: update outdated comment
Date: Tue, 24 Feb 2026 10:28:40 +0800

The function vmemmap_pte_range() was refactored into vmemmap_pte_entry()
by commit fb93ed63345f ("mm: hugetlb_vmemmap: use walk_page_range_novma()
to simplify the code").  Both functions share the key behavior that the
reuse page is identified first before remapping begins.  Update the
comment accordingly.

Link: https://lkml.kernel.org/r/20260224022840.1936-1-kexinsun@smail.nju.edu.cn
Signed-off-by: kexinsun <kexinsun@smail.nju.edu.cn>
Reviewed-by: Muchun Song <muchun.song@linux.dev>
Cc: David Hildenbrand <david@kernel.org>
Cc: Julia Lawall <julia.lawall@inria.fr>
Cc: Oscar Salvador <osalvador@suse.de>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/hugetlb_vmemmap.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/hugetlb_vmemmap.c~mm-hugetlb_vmemmap-update-outdated-comment
+++ a/mm/hugetlb_vmemmap.c
@@ -350,7 +350,7 @@ static int vmemmap_remap_free(unsigned l
 	/*
 	 * In order to make remapping routine most efficient for the huge pages,
 	 * the routine of vmemmap page table walking has the following rules
-	 * (see more details from the vmemmap_pte_range()):
+	 * (see more details from the vmemmap_pte_entry()):
 	 *
 	 * - The range [@start, @end) and the range [@reuse, @reuse + PAGE_SIZE)
 	 *   should be continuous.
_
